content/services/development.md

```diff
@@ -8,10 +8,151 @@ image:
+  - title: Transparent & Interactive
+    text: |
+      We encourage clients to stay closely connected with the daily progress, offering frequent feedback and re-prioritization via Trello or Pivotal Tracker.
+  - title: Accessible & Responsive
+    text: |
+      We use the accessible open web stack: semantic HTML5, unobtrusive JavaScript, and advanced CSS – backed by Python & Django on the server.
 summary: |
-  **We integrate design and development in an agile process**,
-  so you only need to hire one team --
-  completely focused on achieving your goals
-  with smooth and efficient progress
-  from concept to launch and beyond.
+  **We integrate design and development in an agile process**, so you only need to hire one team – completely focused on achieving your goals with smooth and efficient progress from concept to launch and beyond. Starting from user needs, we collaborate with you to deliver usable features quickly and efficiently.
+
+  We provide expertise across a range of technologies, including Python/Django, accessible HTML, modern Sass/CSS, Vue, Node, Backbone/Marionette, and more.
 ---
+
+{% import 'embed.macros.njk' as embed %}
+{% import 'quotes.macros.njk' as quotes %}
+{% import 'layout.macros.njk' as layout %}
+
+{{ quotes.find(
+  collections.all,
+  slugs=['investment', 'curious']
+) }}
+
+{{ layout.title('Services') }}
+
+{% call embed.media_block(
+  media=embed.screen(
+    content=embed.img(
+      src='pages/trig-phone.jpg',
+      alt='trig race screen',
+      width=744 ,
+      height=1611,
+      sizes='media'
+    )
+  )
+) %}
+
+### Design
+
+- User Experience (UX) - OOUX Certified
+- User Interface (UI) & Interaction
+- Logos & Branding
+- Integrated Design Systems
+- Usability Testing
+- Information Architecture
+
+### Development
+
+- Full-stack Frontend & Backend
+- 100% Test Coverage
+- Full Documentation
+- Automated Pattern Libraries
+- WCAG Accessibility
+- Refactors & Performance
+
+{% endcall %}
+
+{{ quotes.find(
+  collections.all,
+  slugs=['investment', 'curious']
+) }}
+
+{{ layout.title('Experience') }}
+
+{% call embed.media_block(
+  media=embed.screen(
+    content=embed.img(
+      src='pages/medcurbside-phone.jpg',
+      alt='medcurbside welcome screen',
+      width=758 ,
+      height=1640,
+      sizes='media'
+    )
+  )
+) %}
+
+### Languages & Platforms
+
+- Python **with** Django (*core contributor*)
+- HTML/CSS **with** Sass (*core contributor*)
+- JavaScript **with** Vue & React
+- Backbone/Marionette
+- Node
+- WordPress
+
+### Client Industries
+
+- Educators
+- HIPAA-compliant wellness providers
+- Non-profits
+- Community organizations
+- Professional gear manufacturers
+- Startups & enterprise companies
+- Urban planners
+- Data analysts
+- Software companies
+- Theaters, writers, & other artists
+
+{% endcall %}
+
+## Our Process Is ...
+
+
+{{ layout.grid(
+  name='special-bg',
+  content=process
+) }}
```

Comment:
@mirisuzanne I wonder if I should be using teh offset grid class on this layout.grid item, but if I do, then the inner contents also need offset-item applied and I don't think I am able to do that from this .md page?

Reply:
I don't think we need an offset-grid here? We use that for layouts like the work samples page, and for multiple quotes. This grid looks great as-is, except for some cleanup:

- needs padding (`var(--spacer) var(--page-margin)`?) on the special-background container
- needs the headline inside the container, which might mean…
- we want to separate the special-background container from the process grid? (Maybe also give it a more meaningful name?)

Overall, I think this is coming together nicely. But I'm happy to talk through any other questions you have.